runtime/doc/lsp.txt

```diff
@@ -762,6 +762,11 @@ start_client({config})                                *vim.lsp.start_client()*
                     {init_options}     Values to pass in the initialization
                                        request as `initializationOptions` .
                                        See `initialize` in the LSP spec.
+                    {settings}         User-specified settings to be passed to 
```

Comment:
The settings are now defined twice. Once for the `workspace/configuration` and once for `workspace/didChangeConfiguration`. I think they should be merged into something like this:

```
                    {settings}         Optional map with language server specific
                                       settings. These are returned to the language server if
                                       requested via `workspace/configuration`. Keys are
                                       case-sensitive.

                                       If they are not `nil`, then the client
                                       will also send an initial
                                       `workspace/didChangeConfiguration`
                                       notification to the server.
```

Reply:
I think we should also probably mention it happens after on_init since I've seen people edit settings within the on_init function.